.github/workflows/main.yaml

```diff
@@ -349,3 +351,86 @@ jobs:
         run: |
           export DOCKER_BUILD_AND_SYSTEM_PRUNE="true"
           make image 
+  container_runtime_test:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        container-runtime: [ 'containerd', 'cri-o', 'docker' ]
```

Comment:
we don't need add containerd here, because e2e tests test it.

Reply:
ok